Gemfile

```diff
@@ -15,3 +15,4 @@ gem "pry-byebug"
 gem "better_errors"
 gem "binding_of_caller"
 gem "awesome_print"
+gem 'simplecov', require: false, group: :test
```

Comment:
__[RuboCop]__

Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.


<!-- Comment by [Sider](https://sider.review) -->